server/src/dao/CustomCardRepository.js

```diff
@@ -46,4 +46,30 @@ export class CustomCardRepository extends BaseRepository {
 
         return cards;
     }
+
+    async findMyCardsCountsByUserId({ startDate, endDate, userId }) {
+        const cardCountList = this.createQueryBuilder('card')
```

Comment:
```suggestion
        const cardCountList = await this.createQueryBuilder('card')
```
await이 빠진거 같아요! 테스트가 잘 수행 되었나요? 저번에 동훈님 await 안 쓰셨었는데 잘 됐다고 하시더라구요. 궁금하네요.

Reply:
테스트하는데 문제는 없었습니다! 그래서 PR도 보냈던 거구요.
이제보니 getMyCardCountByPeriod() 서비스 함수에서도 await을 사용하지 않았는데 성공이 되었더라구요.
제 생각에는 서비스나 라우터쪽에서 async await을 해주니 dao에서 프로미스 객체를 반환해주어도 정상적으로 실행이 됬던 것 같아요. 🤔
근데 의미적으로 await을 추가해주는게 좋을것 같다는 생각이 드네요.
commit ca19ab5 수정했습니다!

Reply:
신기하네요... 왜 이런 현상이 일어날까요? await을 안 쓰면 pending 상태가 될텐데...
일단 알겠습니다!